dot/parachain/collator-protocol/message.go

```diff
@@ -1,18 +1,22 @@
 // Copyright 2023 ChainSafe Systems (ON)
 // SPDX-License-Identifier: LGPL-3.0-only
 
-package parachain
+package collatorprotocol
 
 import (
+	"errors"
 	"fmt"
 
 	"github.com/ChainSafe/gossamer/dot/network"
 	parachaintypes "github.com/ChainSafe/gossamer/dot/parachain/types"
+	"github.com/ChainSafe/gossamer/dot/peerset"
 	"github.com/ChainSafe/gossamer/lib/common"
 	"github.com/ChainSafe/gossamer/pkg/scale"
 	"github.com/libp2p/go-libp2p/core/peer"
 )
 
+const LEGACY_COLLATION_PROTOCOL_V1 = "/polkadot/collation/1"
```

Comment:
just for code consistency

```suggestion
const LegacyCollactionProtocolV1 = "/polkadot/collation/1"
```